.github/workflows/ci-dgraph-tests-arm64.yml

```diff
@@ -60,16 +54,13 @@ jobs:
       - name: Run Unit Tests
         run: |
           #!/bin/bash
-          # clean cache
-          go clean -testcache
           # go env settings
           export GOPATH=~/go
           # move the binary
-          cp dgraph/dgraph ~/go/bin 
-          # build the test binary
-          cd t; go build .
```

Comment:
the `go build .` is needed right?

Reply:
We are calling it in the previous step (around line 50)

Reply:
+1

Reply:
replicate to all yaml?